### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ws-ckina**
Here is the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# git-exploring
```
Comment (ws-ckina) [inline README.md:1]: comment.1

Comment (ckinan) [reply README.md:1]: comment.2

**Comment by ckinan** [comment]
comment.3

File: README.md (hunk 1 above)
Comment (ckinan) [reply README.md:1]: comment.3

**Comment by ckinan** [comment]
comment.4

File: README.md (hunk 1 above)
Comment (ckinan) [reply README.md:1]: comment4

Comment (ckinan) [reply README.md:1]: comment.5

**Pull request merged**